nilearn/plotting/surf_plotting.py

```diff
@@ -527,8 +527,8 @@ def _plot_surf_matplotlib(coords, faces, surf_map=None, bg_map=None,
                           alpha='auto', bg_on_data=False,
                           darkness=.7, vmin=None, vmax=None, cbar_vmin=None,
                           cbar_vmax=None, cbar_tick_format='%.2g',
-                          title=None, title_font_size=18, output_file=None,
-                          axes=None, figure=None, **kwargs):
+                          title=None, output_file=None,
+                          axes=None, figure=None):
```

Comment:
both are unused

Reply:
Sounds reasonable. We can add them back later if we find time for that.

Reply:
yes I agree: this PR in fact just tries to align the doc string and function / method "declaration" with what the code actually does (or can be expected to do).

also some functions having `**kwargs` actually lead to some tests with arguments used but that did nothing